doc/rst/platforms/macosx.rst

```diff
@@ -57,3 +56,48 @@ section of :ref:`chapelhome-quickstart` for next steps.
 
 
 .. _Homebrew: https://brew.sh/
+
+
+--------------------
+Building from Source
+--------------------
+
+If you are not using the Homebrew package, it will be necessary to
+install a few development tools in order build Chapel. The instructions
+below show one way of getting a working development environment.
+
+First, install XCode_ or the developer Command Line Tools. XCode can be
+installed from the App store.
```

Comment:
I'm not sure that installing XCode from the App Store will expose the command line tools like clang on PATH. Perhaps it's worth mentioning `xcode-select --install` (which, paradoxically, does not actually require XCode)

Reply:
It does do that based on my recent experience (this past weekend) on a fresh system. Nonetheless I am happy to mention xcode-select --install.